<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueDialog</class>
 <widget class="QDialog" name="QueueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="elemValueSpin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addElemButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="popButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dequeue</string>
   </property>
  </widget>
  <widget class="QLabel" name="popLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count</string>
   </property>
  </widget>
  <widget class="QLabel" name="countLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
